### PR TITLE
Mack inclinometer

### PR DESCRIPTION
**Pull request by SolbiatiAlessandro**
we need to fix this

### CODE REVIEW

File: app/src/main/java/com/selfdrivingboat/boatcontroller/MainActivity.java [hunk 1]
```diff
@@ -1,394 +1,445 @@
...
+    private Sensor mACCELEROMETER;
+    public float  acceleRometer_x,  acceleRometer_y,  acceleRometer_z;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        sensorManager = (SensorManager) getSystemService(SENSOR_SERVICE);
+        mACCELEROMETER = sensorManager.getDefaultSensor(Sensor.TYPE_ACCELEROMETER);
+        sensorManager.registerListener(this,mACCELEROMETER, SensorManager.SENSOR_DELAY_NORMAL);
```
Comment (SolbiatiAlessandro) [inline app/src/main/java/com/selfdrivingboat/boatcontroller/MainActivity.java]: this lines breaks the app.
If you comment it out you will see that everything works fine, if you leave it the app break. 
can't find any stack trace just find "BoatController isn'responding"

wdyt @mack-4567 ?

**Comment by SolbiatiAlessandro** [comment]
Fixed "app not responsive" in https://github.com/theselfdrivingboat-com/BoatController/pull/9/commits/5ac151d8f0bc396027e1eb53bc3c13fc46cc3537, left some follow up for @mack-4567  to do there in commit message. Send update to this PR with those changes and we merge

Screenshot 2021-06-19 at 21 15 44
Screenshot 2021-06-19 at 21 15 09

**Comment by SolbiatiAlessandro** [comment]
hey @siddharth2010  can I get your opinion on https://github.com/theselfdrivingboat-com/BoatController/pull/9/commits/5ac151d8f0bc396027e1eb53bc3c13fc46cc3537 if we are getting this right?

@mack-4567 implemented this phone accelerometer listener but the app was getting unresponsive. My guess was that it was getting triggered to often so I 1) got it triggered less often 2) put it in async task like you explained me. This looks like is fixing then problem.

Does this make sense to you?

**Comment by ghost** [comment]
Hey alex! Just had a look at the commit and I think it looks great! ✌️

**Comment by SolbiatiAlessandro** [comment]
@mack-4567 here is what is left to do to merge this PR 

from latest commit 7e9cee5
- [ ] (First in code TODO) run this and make sure that the inclination is correct
if inclination is not correct change code with magnetometer reading, I
already added them here if we end up needing them.

NOTE: if the app is breaking find and report us the error here

- [ ] (Third in code TODO) send all data to influxdb
check if the data come, or just add them to the function

from previous commit 5ac151d
- [ ] open the Logcat and filter logs with "androidAccellerometer", check
  how often we send the logs to the server: we still send them too
often! set the parmatere so that we send them every 2 minutes

When these three are done we can merge and have the inclinometer on the app!

**Comment by SolbiatiAlessandro** [comment]
We are following this here https://github.com/theselfdrivingboat-com/BoatController/pull/11

**Pull request closed without merging**